modules/event.js

```diff
@@ -0,0 +1,22 @@
+// Function to handle navigation links click event
+
+function handleNavigationLinkClick(event) {
```

Comment:
- Hey, please, kindly use the ES6 arrow function since you're working on learning to use the ES6 syntax.